docs/spark-qualification-tool.md

```diff
@@ -6,12 +6,24 @@ nav_order: 8
 
 # Qualification Tool
 
-The Qualification tool analyzes Spark events generated from CPU based Spark applications to determine 
-if the RAPIDS Accelerator for Apache Spark might be a good fit for GPU acceleration.
+The Qualification tool analyzes Spark events generated from CPU based Spark applications to help quantify
+the expected acceleration of migrating a Spark application to GPU.
+
+The tool first analyzes the operators executed from the CPU log and determines which ones would be qualified to run on
+GPU.  
+The tool then uses estimates from historical queries and benchmarks to estimate a speedup at an individual operator
+level to calculate how much a specific operator would accelerate on GPU for the specific application.  
+To calculate the _"Estimated GPU App Duration"_, it will add up the accelerated operator durations along with durations
+that could not run on CPU because they are unsupported operators or not SQL operations.
 
 This tool is intended to give the users a starting point and does not guarantee the
-applications with the highest scores will actually be accelerated the most. Currently,
+applications with the highest  _recommendation_ will actually be accelerated the most. Currently,
 it reports by looking at the amount of time spent in tasks of SQL Dataframe operations.
+
+> **Disclaimer!**  
+> Estimates provided by the Qualification tool are given assuming supported data formats  
+> and expressions are used in the application.
```

Comment:
put a link to see https://nvidia.github.io/spark-rapids/docs/supported_ops.html.  

I also think either here or above we should explicitly state the tool is using supported SparkPlan or Executor Nodes and does not yet look at the expressions or datatypes

Reply:
Done!